packages/wasm-edit/test/replace-node.js

```diff
@@ -139,7 +139,7 @@ describe("replace a node", () => {
     const newBinary = edit(actualBinary, {
       Instr(path) {
         if (path.node.id === "get_global") {
-          const newNode = t.callIndirectInstructionIndex(t.indexLiteral(2));
+          const newNode = t.callIndirectInstructionIndex(t.indexLiteral(0));
```

Comment:
I'm not sure `callIndirectInstructionIndex` should actually exist! I think our AST representation for call_indirect should always be populated with params / return - avoiding the need for different AST representations when parsing WASM vs. WAT